lib/tasks/find_possible_duplicate_accounts.rake

```diff
@@ -0,0 +1,64 @@
+namespace :accounts do
+  namespace :duplicate_accounts do
+    desc "Find possible duplicate acccounts"
+    task :find => [:environment] do
+      CSV.open("duplicate_users_by_name.csv", 'w+') do |csv|
+        csv << ['User First Name', 'User Last Name', 'Email Address(es)', 'User ID', 'Social Signup Successful', 'Reset Password Help Requested', 'Help Request Failed']
+
+        where_names_match = User.joins{ User.unscoped.as(same)
+                        .on{
+                          (same.id != ~id) & ((lower(same.first_name) == lower(~first_name)) & (lower(same.last_name) == lower(~last_name)))
+                        }
+                      }.uniq.order(id: :asc)
+
+        where_names_match.find_each do |user|
+          csv << ["#{user.first_name}",
+                  "#{user.last_name}",
+                  "#{user.contact_infos.map(&:value).join(", ") if user.contact_infos.any?}",
+                   user.id,
+                   sign_up_successful(user),
+                   help_requested(user),
+                   help_request_failed(user)
+                 ]
+        end
+      end
+
+      CSV.open("duplicate_users_by_email.csv", 'w+') do |csv|
+        csv << ['Email Address', 'ContactInfo ID', 'User First Name', 'User Last Name', "User ID", 'Social Signup Successful', 'Reset Password Help Requested', 'Help Request Failed']
+
+        where_email_addresses_match = ContactInfo.joins{ ContactInfo.unscoped.as(same)
+                                                   .on{ (same.id != ~id) & (lower(same.value) == lower(value)) }
+                                        }.joins{ user.outer }.preload(:user).uniq
+
+        where_email_addresses_match.find_each do |contact_info|
+          csv << [contact_info.value,
+                  contact_info.id,
+                  "#{contact_info.user.try(:first_name)}",
+                  "#{contact_info.user.try(:last_name)}",
```

Comment:
It seems like there is no need for the extra quoting work; if the strings exist, they'll be strings, if not, `nil` will convert to an empty string.